docs/interfaces/webui-if.rst

```diff
@@ -4,12 +4,93 @@
-established. After logging in, you are redirected to your initial URL.
+To access the WebUI, go to the default URL ``http://master-addr:8080`` in your web browser, where
+``master-addr`` is the hostname or IP address of the master. If Transport Layer Security (TLS) is
+enabled, use port ``8443`` instead.
 
-Click **Sign Out** at the top-right to end the Determined session.
+If you have not yet established a valid Determined session, the WebUI will automatically redirect
+you to the login page. Once you log in, you will be redirected to the initial URL you entered.
+
+To end the Determined session and log out of the WebUI, click **Sign Out**.
+
+************************
+ Creating an Experiment
+************************
+
+If you have an existing experiment or trial, you can use the WebUI to create an experiment. To find
+out more, visit :ref:`experiments`.
+
+**********************************************************
+ Organizing Your Experiments into Projects and Workspaces
+**********************************************************
+
+The WebUI lets you organize your experiments into projects and workspaces. To get started, visit
+:ref:`workspaces`.
+
+*******************************************
+ Configuring the Behavior of an Experiment
+*******************************************
+
+Many experiment configuration settings can be performed via the WebUI. To find out more, visit
+:ref:`experiment-config-reference`.
+
+***************************************
+ Evaluating the Performance of a Model
+***************************************
+
+You can access training and validation performance information via the WebUI. To see model
+evaluation in action, follow the steps described in the :ref:`pytorch-mnist-tutorial`.
+
+*************************************
+ Adding Models to the Model Registry
+*************************************
+
+You can use the WebUI to create and edit models and add models to the model registry. You can also
+use the WebUI to edit model metadata. To find out more, visit :ref:`organizing-models`.
+
+***************************************
+ Viewing Historical Cluster Usage Data
+***************************************
+
+You can use the WebUI to view :ref:`historical-cluster-usage-data`.
+
+***********************************
+ Managing User Accounts and Groups
+***********************************
+
+The ``admin`` user manages user authentication including creating and managing users. To learn more,
+visit :ref:`users`.
+
+With the Determined Enterprise Edition, you can also create and manage user groups. To learn more,
+visit :ref:`rbac`.
+
+***********************
+ Configuring Telemetry
+***********************
+
+To find what kind of anonymous information the WebUI collects, visit
+:ref:`common-configuration-options`.
+
+************************
+ Managing the Job Queue
+************************
+
+To find out how to view and modify the Job Queue in the WebUI, visit :ref:`experiments`.
```

Comment:
Could this connect to the `#view-the-job-queue` anchor instead?

Reply:
yes, good catch